.gitignore

```diff
@@ -17,3 +17,5 @@ tmp*/
 *.make
 *.temp
 .coverage
+_build
+.DS_Store
```

Comment:
The `.DS_Store` line should go in your global `.gitignore`, not in here.  It's not specific to our repo, it's specific to your dev environment.